src/Servers/Kestrel/Core/src/KestrelConfigurationLoader.cs

```diff
@@ -24,26 +16,24 @@ namespace Microsoft.AspNetCore.Server.Kestrel;
 /// </summary>
 public class KestrelConfigurationLoader
 {
+    private readonly IHttpsConfigurationService _HttpsConfigurationService;
```

Comment:
nit: lower case for fields.
```suggestion
    private readonly IHttpsConfigurationService _httpsConfigurationService;
```

Reply:
I have no idea how that happened in so many places, but I think I caught them all.